compiler/src/iree/compiler/Codegen/Common/TileSizeSelection.h

```diff
@@ -113,6 +113,12 @@ class TilingConfig {
   /// and reduction dimensions.
   SizesAndScalableFlags getVectorTileSizes();
 
+  /// Returns a new `LoweringConfigAttr`, with the tile sizes of vector
+  /// dimensions, set to `sizes`, and the corresponding scalability set to
+  /// `scalableFlags`.
+  IREE::Codegen::LoweringConfigAttr
+  withNewVectorSizes(ArrayRef<int64_t> sizes, ArrayRef<bool> scalableFlags);
```

Comment:
Nit: rename to `getLoweringConfigWithNewVectorSizes` .. wider names are OK by me.